packages/sanes-chrome-extension/src/routes/share-identity/index.dom.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { Store } from 'redux';
+import { RootState } from '../../store/reducers';
+import { aNewStore } from '../../store';
+import { travelToShareIdentity } from './test/travelToShareIdentity';
+import {
+  clickOnAcceptButton,
+  clickOnBackButton,
+  clickOnRejectButton,
+  confirmSelectionButton,
+  checkPermanentRejection,
+} from './test/operateShareIdentity';
+
+describe('DOM > Feature > Share Identity', (): void => {
+  let store: Store<RootState>;
+  let identityDOM: React.Component;
+
+  beforeEach(async () => {
+    store = aNewStore();
+    identityDOM = await travelToShareIdentity(store);
+  });
+
+  it('should accept incoming request and come back', async (): Promise<void> => {
+    await clickOnAcceptButton(identityDOM);
+    await confirmSelectionButton(identityDOM);
+    //TODO: Check here that share request rejection has been reject successfuly
+
+    /**
+     * Remove this code if not required in case if there is another redirection
+     * in confirmAcceptButton method. And apply this method in separate test method.
+     */
+    await clickOnBackButton(identityDOM);
```

Comment:
What are you trying to do here? as far as I see in the invision mock, the final redirection is to the account status view. 🤔

Reply:
Ok, will check for account status view

Reply:
Hmmm.. where to redirect in case of requests queue?

Reply:
I got the answer

> Once the request is dispatched, the view should be redirected to: `request view` if there are more pending requests, or to `account-status view` if nothing is left.